storefront/app/view_models/workarea/storefront/product_view_model.rb

```diff
@@ -1,7 +1,6 @@
 module Workarea
   module Storefront
     class ProductViewModel < ApplicationViewModel
-
```

Comment:
Can remove changes to this file after suggestion above.

Reply:
that would kinda defeat the purpose of all this, because a string isn't `.navigable?`, so we still need to pass the full product model into the breadcrumbs as `:last` in order to generate the right URL.

Reply:
Do the tests pass without the change?

Reply:
fixed it by checking whether `last.class.includes? Navigable` so we don't have to write an additional method.